cucumber/api/features/authenticate.feature

```diff
@@ -21,6 +21,19 @@ Feature: Exchange a role's API key for a signed authentication token
       cucumber:user:alice successfully authenticated with authenticator authn
     """
 
+  Scenario: Authenticate response should be encoded if asked
+    Then I can POST "/authn/cucumber/alice/authenticate?encoded=true" with plain text body ":cucumber:user:alice_api_key"
+    And the HTTP response content type is "text/plain"
+    And the HTTP response content type is base64 encoded
```

Comment:
i would add also a step in the manner of `Then user "alice" has been authorized by Conjur`, and move the logic behind the step above to be similar to what we do there. 

please see the authn-azure tests for reference.

Reply:
Discussed